app/views/groups/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<header>
+<h1 class="new-category-heading"><b>Categories</b></h1>
+</header>
+<% if @categories.empty? %>
+  <h2>There is no categories for now</h2>
+<% end %>
+<div >
+    <ul class="categories-list">
+        <% @categories.each do |category| %>
+          <li class="my-6">
+            <% @amount = 0%>
+          <img src=<%=category.icon%> class="images"  alt="Girl in a jacket" width="80" height="80">
+          <% @total.where(group: category).each do | item | %>
+          <% @amount += item.exchange.amount %>
+          <% end %>
+          <%= link_to category.name, "/groups/#{category.id}" %>
+          <p> $<%= @amount%></p>
+          </li>
+        <% end %>
+    </ul>
+</div>
+<div class="new-category button-div">
+  <%= link_to 'Add New Category', new_group_path %>
+</div>
```

Comment:
- [ ] You did a great job on the project requirements but you need to fix the below design issues to match with the given design.
  - Please make sure the styles for the "Add new category" and "Add new transaction" buttons match with the given design. (Kindly also use the correct color `#5fb523`).
  
### Your design
  
![image](https://github.com/tchasinga/Budget-app-manager/assets/83803414/08cd5e24-8fbf-4848-911b-828e3a48a73b)
![image](https://github.com/tchasinga/Budget-app-manager/assets/83803414/4cd15c10-38fe-4ba4-9dca-003a8c28b2d8)

### Given design
  
![image](https://github.com/tchasinga/Budget-app-manager/assets/83803414/604d1f59-9b8d-46fa-b014-abf82a50e402)